custom/icds_reports/ucr/data_sources/cases_ag_care.json

```diff
@@ -0,0 +1,393 @@
+                "type": "property_name",
+                "property_name": "migration_status"
+              },
+              "type": "related_doc",
+              "related_doc_type": "CommCareCase",
+              "doc_id_expression": {
+                "type": "named",
+                "name": "parent_id"
+              }
+            },
+            "type": "boolean_expression",
+            "property_value": "migrated",
+            "comment": null
+          },
+          "expression_if_true": {
+            "type": "constant",
+            "constant": 1
+          },
+          "expression_if_false": {
+            "type": "constant",
+            "constant": 0
+          }
+        },
+        "type": "expression",
+        "column_id": "is_migrated",
+        "is_nullable": true,
+        "datatype": "integer",
+        "is_primary_key": false,
+        "create_index": false,
+        "transform": {},
+        "display_name": null,
+        "comment": null
+      },
+      {
+        "expression": {
+          "type": "conditional",
+          "test": {
+            "operator": "not_eq",
+            "expression": {
+              "value_expression": {
+                "datatype": null,
+                "type": "property_name",
+                "property_name": "registered_status"
+              },
+              "type": "related_doc",
+              "related_doc_type": "CommCareCase",
+              "doc_id_expression": {
+                "type": "named",
+                "name": "parent_id"
+              }
+            },
+            "type": "boolean_expression",
+            "property_value": "not_registered",
+            "comment": null
+          },
+          "expression_if_true": {
+            "type": "constant",
+            "constant": 1
+          },
+          "expression_if_false": {
+            "type": "constant",
+            "constant": 0
+          }
+        },
+        "type": "expression",
+        "column_id": "is_availing",
+        "is_nullable": true,
+        "datatype": "integer",
+        "is_primary_key": false,
+        "create_index": false,
+        "transform": {},
+        "display_name": null,
+        "comment": null
+      }
+    ],
+    "named_expressions": {
+      "parent_id": {
+        "type": "property_name",
+        "property_name": "host_person_id",
+        "datatype": null
+      }
+    },
+    "named_filters": {
+      "open_cases": {
+        "operator": "eq",
+        "expression": {
+          "datatype": null,
+          "type": "property_name",
+          "property_name": "closed"
+        },
+        "type": "boolean_expression",
+        "property_value": false,
+        "comment": null
+      }
+    },
+    "engine_id": "icds-ucr-citus",
+    "sql_settings": {
+      "citus_config": {
+        "distribution_type": "hash",
+        "distribution_column": "supervisor_id"
+      },
+      "primary_key": ["supervisor_id","doc_id"]
+    },
+    "sql_column_indexes": [],
```

Comment:
Not sure what to use here. @calellowitz any thoughts?

Reply:
Hey, I will let Cal give his stand point. But I can jump in. This field is used to create the indexes in the UCR. You can see the examples here:https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/child_health_cases.json#L1641.

Usually, we create the indexes as per how the ucr will be used. For example: if you know that queries on this table will be like `select column from ucr where state_id='x' and opened_on>'y-date'` then good to create an index on state_id and opened_on for its better performance.  

@orangejenny

Reply:
Added a `(awc_id, opened_on)` index in 12998474580 because that's how the [one UCR](https://india.commcarehq.org/a/icds-cas/reports/configurable/2c482a1269f8239768b9c6bc251dfaad/) that depends on this data source groups its rows.